infrastructure/docker/services/php-base/Dockerfile

```diff
@@ -7,6 +7,11 @@ RUN apt-get update \
     && echo "deb https://packages.sury.org/php buster main" > /etc/apt/sources.list.d/sury.list \
     && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B188E2B695BD4743
 
+ARG NODEJS_VERSION=12.x
+RUN apt-get update \
```

Comment:
* Why do you need nodejs in the PHP container?
* More over, it's weird to install nodejs with 2 differents way (via apt in the builder, and here)